### ISSUE TITLE
withGlobal cannot find the global state in React Native.

### ISSUE THREAD

**Issue by Brianop** [CharlesStover/reactn#78]
Hi, im trying to use reactn on react native but im getting this issue when using the withGlobal HOC

![WhatsApp Image 2019-05-17 at 9 40 29 AM](https://user-images.githubusercontent.com/15720472/57935649-e5a19200-7887-11e9-960d-7698d1089d19.jpeg)

```
import React, {withGlobal} from 'reactn'

const GroupSelector = () => {
    return (
        ...
    );
}

export default withGlobal(
    global => ({
        groups: global.groups
    }),
)(GroupSelector);
```

Any assistance you can provide would be greatly appreciated

**Comment by CharlesStover** [comment]
Thanks for reporting this. What version of React are using? Is it < 16.3?

**Comment by Brianop** [comment]
Im using "react": "16.5.0"

**Comment by CharlesStover** [comment]
Are you sure you are importing `withGlobal` from `'reactn'` and not `'reactn/build/with-global'`? I've had other reports that Webstorm's auto-imports are the incorrect files.

**Comment by BDQ** [comment]
I encountered the same issue, importing `withGlobal` from `reactn` for sure.

**Comment by CharlesStover** [comment]
Awesome. I am able to reproduce this. I'll try to get it tackled today in 2.0.4.

**Comment by CharlesStover** [comment]
This is fixed and will be deploying a few moments.

As a part of this fix, I've made a breaking change that should be have been included in `2.0.0`.

The getter and setter functions previously accepted two parameters: the global state and the HOC's props. `(global, props) => {}`.

Now, they accept three parameters: the global state, the dispatchers for your reducers, and the HOC's props. `(global, dispatch, props) => {}`.

If you are relying on `props` as your second parameter, be sure to change it to the third parameter by adding `dispatch` as the second.

before:
```JavaScript
export default withGlobal(
  (global, props) => ({ ... }),
  (setGlobal, props) => ({ ... }),
)(MyComponent);
```
after:
```JavaScript
export default withGlobal(
  (global, dispatch, props) => ({ ... }),
  (setGlobal, dispatch, props) => ({ ... }),
)(MyComponent);
```

**Comment by BDQ** [comment]
Thanks @CharlesStover
